Tests/XMTPTests/ClientTests.swift

```diff
@@ -492,4 +492,41 @@ class ClientTests: XCTestCase {
 		let newState = try await alixClient3.inboxState(refreshFromNetwork: true)
 		XCTAssertEqual(newState.installationIds.count, 1)
 	}
+
+	func testSignMessage() async throws {
+		let key = try Crypto.secureRandomBytes(count: 32)
+		let alix = try PrivateKey.generate()
+		let options = ClientOptions.init(
+			api: .init(env: .local, isSecure: false),
+			   enableV3: true,
+			   encryptionKey: key
+		   )
+
+		let alixClient = try await Client.create(
+			account: alix,
+			options: options
+		)
+		let privateKey = try PrivateKey(alixClient.keys.identityKey)
+
+		let stringToSign1 = "TEST_STRING_TO_SIGN"
+		
+		let signature1_1 = try await privateKey.sign(stringToSign1.data(using: .utf8)!)
+		let uint1_1 = try [UInt8](signature1_1.serializedData())
+
+		let signature1_2 = try await privateKey.sign(stringToSign1.data(using: .utf8)!)
+		let uint1_2 = try [UInt8](signature1_2.serializedData())
+
+		XCTAssertEqual(uint1_1, uint1_2)
+
+		let stringToSign2 = "short"
```

Comment:
at first I thought it was because the string was short but I managed to make it fail with longer strings also… maybe a byte length thing?